test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1

```diff
@@ -0,0 +1,111 @@
+﻿Describe "ConvertTo-Html DRT Unit Tests" -Tags "CI" {
+    $customObject = [pscustomobject]@{"Name" = "John Doe"; "Age" = 42; "Friends" = ("Jack", "Jill")}
+    $newLine = [System.Environment]::NewLine
```

Comment:
This test code should go within BeforeAll instead of existing within the Describe. See [Pester Do's and Don'ts](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/PesterDoAndDont.md)